templates/zerver/integrations.html

```diff
@@ -2062,6 +2070,85 @@
 
     <div id="trello" class="integration-instructions">
 
+      <p>
+        First, create the stream you'd like to use for Trello notifications,
+        and subscribe all interested parties to this stream. We recommend the
+        name <code>trello</code>.
+      </p>
+
+      <p>
+        Your webhook URL is:<br/>
+        <code>https://api.zulip.com/v1/external/trello?api_key=abcdefgh&stream=trello</code><br/>
+        where <code>api_key</code> is the API key of your Zulip bot,
+          and <code>stream</code> is the stream name you want the notifications sent to.
+      </p>
+
+      <p>
+        Before you'll be able to create a Webhook,
+        you need to follow steps below to get an <b>APPLICATION_KEY</b> and a <b>UserToken</b>.
+        Also we need to fetch the <b>idModel</b> of the board.
+      </p>
+
+      <p>
+        To generate the <b>APPLICATION_KEY</b> open this URL in your web browser:<br/>
+        <code>https://trello.com/1/appkey/generate</code>.
+      </p>
+
+      <p>
+        To generate read access token, fill in and open this URL in the browser while being logged into your Trello account:
```

Comment:
Better:

`To generate a read access token, fill in and open this URL in your browser while logged into your Trello account:`